src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -1473,6 +1497,29 @@ private static BundleEntryComponent allergy(RandomNumberGenerator rand,
     Code code = allergy.codes.get(0);
     allergyResource.setCode(mapCodeToCodeableConcept(code, SNOMED_URI));
```

Comment:
We can assume `SNOMED_URI` should be used here, since a drug allergy/intolerance could instead require using `RXNORM_URI`. Some more logic is required here.

Reply:
I think this is only used as a default in case the provided `code` doesn't contain a system for some reason. I was looking at some FHIR resources and the actual URI used was RxNorm when appropriate, ex:
```
{
      "resourceType": "AllergyIntolerance",
      "id": "4214370a-a980-d3a6-ecb2-d601c1423848",
      "clinicalStatus": {
        "coding": [ {
          "system": "http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical",
          "code": "active"
        } ]
      },
      "verificationStatus": {
        "coding": [ {
          "system": "http://terminology.hl7.org/CodeSystem/allergyintolerance-verification",
          "code": "confirmed"
        } ]
      },
      "type": "allergy",
      "category": [ "medication" ],
      "criticality": "low",
      "code": {
        "coding": [ {
          "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
          "code": "7984",
          "display": "Penicillin V"
        } ],
        "text": "Penicillin V"
      },
      "patient": {
        "reference": "urn:uuid:e27548c2-8384-8208-17dc-aeb0587bdecc"
      },
      "recordedDate": "1997-12-15T03:30:39-05:00",
      "reaction": [ {
        "manifestation": [ {
          "coding": [ {
            "system": "http://snomed.info/sct",
            "code": "247472004",
            "display": "Wheal (finding)"
          } ],
          "text": "Wheal (finding)"
        } ],
        "severity": "mild"
      }, {
        "manifestation": [ {
          "coding": [ {
            "system": "http://snomed.info/sct",
            "code": "271807003",
            "display": "Eruption of skin (disorder)"
          } ],
          "text": "Eruption of skin (disorder)"
        } ],
        "severity": "mild"
      } ]
    },
```

Reply:
Ah, you are correct. `mapCodeToCodeableConcept` only uses that system by default if the code doesn't already have one.